app/controllers/goldencobra/sessions_controller.rb

```diff
@@ -54,5 +50,14 @@ def logout
 
     def register
     end
+
+    private
+
+    def usermodel
+      return false if params[:usermodel].blank?
+      return false unless %w(user visitor).include?(params[:usermodel].to_s.downcase)
```

Comment:
Keine bekannt? Goldencobra ist doch eigentlich ein öffentliches Gem, das in allen möglichen Apps eingesetzt werden könnte. Eine künstliche Beschränkung auf diese 2 Models fände ich sehr unvorteilhaft. Es gibt doch genug andere Modelname wie "Admin", "Member" und co die ebenfalls Devise nutzen könnten.